dictionaries/cs.dict.authent-token.php

```diff
@@ -37,6 +37,8 @@
 	'Class:PersonalToken/Attribute:scope/Value:REST/JSON+' => '/webservices/rest.php~~',
 	'Class:PersonalToken/Attribute:scope/Value:Synchro' => 'Synchro~~',
 	'Class:PersonalToken/Attribute:scope/Value:Synchro+' => '/synchro/synchro_import.php and /synchro/synchro_exec.php~~',
+	'Class:PersonalToken/Attribute:scope/Value:CRON' => 'CRON~~',
+	'Class:PersonalToken/Attribute:scope/Value:CRON+' => '/webservices/cron.php~~',
```

Comment:
I think you can safely remove the `~~` from all the `CRON+` dicts, as the path doesn't have translations :wink:

```suggestion
	'Class:PersonalToken/Attribute:scope/Value:CRON+' => '/webservices/cron.php',
```

Reply:
Thanks for the feedback @Hipska , I updated the translations